test/functional/test-3p-environment.js

```diff
@@ -35,6 +35,10 @@ describe('3p environment', () => {
     });
   });
 
+  afterEach(() => {
+    testWin = null;
```

Comment:
> From my reading it doesn't seem like that would actually work memory-wise

It's essentially the [fix they used](https://github.com/mochajs/mocha/pull/2037/files#diff-b3b53682a18f203ac8d29b0e277cad26R749) in v2.4.0.

> But it would not work with --watch, right?

No idea. Mocha's already deleting `beforeEach`/`afterEach` when the describe blocks end.